README.md

```diff
@@ -0,0 +1,15 @@
+Chummer
+-------
+
+"Chummer is a character generator for Shadowrun 4th Edition.
+Not only can you create your character quickly and easily,
+but you can also use Chummer during your character's shadowrunning career,
+to accurately track your Karma, Nuyen, ammo, and everything else all in one place.
+Chummer also includes support for a number of optional rules and house rules and even includes
+support for critters and is useful for players and Game Masters alike!
+It also supports four languages: English, French, German, and Japanese.
+
+Chummer is continuously updated to address bugs and introduce new and interesting features suggested by the community.
+If you have a bug to report or idea to suggest, please report them on the Dumpshock forums by using the link in the menu."
```

Comment:
Can you change the last paragraph to the Issues section of the project instead of the Dumpshock forums?